OneBlink.SDK.Tests/FormsAppsClientTests.cs

```diff
@@ -112,6 +112,15 @@ public async void can_crud_forms_app()
             FormsApp updatedFormsApp = await formsAppsClient.Update(retrievedFormsApp);
             Assert.Equal(updatedName, updatedFormsApp.name);
 
+            // create and delete user
+            FormsAppUser newUser = new FormsAppUser();
+            newUser.formsAppId = updatedFormsApp.id;
+            newUser.email = "test@oneblink.io";
```

Comment:
I assume this actually creates a user, which will send a real email, which will bounce. Do we need to worry about sending emails that will bounce over and over again every time the tests run?

Reply:
Existing inbox I had created for email testing :)